app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -69,6 +68,8 @@ def index
         instance_ids = instances.pluck(:library_instance_id).reject(&:blank?)
         instance_ids += instances.where(:library_instance_id => nil)
         repositories = Repository.where(:id => instance_ids)
+      elsif params[:library] && params[:content_view_version_id] && params[:organization_id]
```

Comment:
Is organization_id required? CVV ID is going to be unique across the application.